src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellNavBarAppearanceTracker.cs

```diff
@@ -85,7 +85,8 @@ public virtual void SetHasShadow(UINavigationController controller, bool hasShad
 
 		#endregion
 
-		[SupportedOSPlatform("ios13.0")]
+		[SupportedOSPlatform("ios15.0")]
+		[SupportedOSPlatform("tvos15.0")]
```

Comment:
I notice a version change... Is this a bug in the macios repo because it should not have compiled?

Reply:
looks like UINavigationBarAppearance was introduced in iOS13, so looks like a MAUI issue and this should've been called on iOS13?

Reply:
JFYI: The warnings found were all for version 13:

```log
'UINavigationBar.StandardAppearance.set' is   only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvos'   13.0 and later.
'UINavigationBarAppearance' is only   supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvos' 13.0   and later.
'UIBarAppearance.ConfigureWithOpaqueBackground()'   is only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later,   'tvos' 13.0 and later.
'UIBarAppearance.BackgroundColor' is only   supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvos' 13.0   and later.
'UINavigationBarAppearance.TitleTextAttributes'   is only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later,   'tvos' 13.0 and later.
'UINavigationBar.ScrollEdgeAppearance' is   only supported on: 'ios' 13.0 and later, 'maccatalyst' 13.0 and later, 'tvos'   13.0 and later.
```